org.mwc.cmap.legacy/src/MWC/GUI/JFreeChart/ColouredDataItem.java

```diff
@@ -69,16 +69,21 @@
 	 */
 	private static final long serialVersionUID = 1L;
 
+	/**
+	 * WARNING. DON'T ADD THE FINAL MODIFIER TO THE FOLLOWING FIELDS.
+	 * https://github.com/debrief/debrief/issues/4861
+	 * Saul 
+	 */
 	/**
 	 * the color for this item
 	 */
-	private final Color _myColor;
+	private Color _myColor = null; // do not make final, it will xstream serialization
```

Comment:
@saulhidalgoaular  - I think we need `break` between `will` and `xstream` :-)

Reply:
Fixed :)